StitchCoreServicesMongoDbRemote/Sources/StitchCoreServicesMongoDbRemote/Internal/CoreRemoteMongoCollection.swift

```diff
@@ -0,0 +1,283 @@
+import Foundation
+import MongoSwift
+import StitchCore
+
+public class CoreRemoteMongoCollection<T: Codable> {
```

Comment:
[nit][minor] All of the operation functions in this class should have labels for their arguments.